packages/shared/friends/sagas.ts

```diff
@@ -435,6 +427,70 @@ export function getFriendRequests(request: GetFriendRequestsPayload) {
   getUnityInstance().AddFriendRequests(addFriendRequestsPayload)
 }
 
+export function getUnseenMessagesByUser() {
+  const client: SocialAPI | null = getSocialClient(store.getState())
+  if (!client) return
+
+  const conversations = client.getAllCurrentConversations()
+
+  const conversationsWithMessages = conversations.filter((conv) => conv.conversation.hasMessages)
+  const unseenPrivateMessages = {}
+
+  for (const conversation of conversationsWithMessages) {
+    unseenPrivateMessages[conversation.conversation.userIds![1]] = conversation.conversation.unreadMessages?.length || 0
```

Comment:
Is the index `1` always used for the other person or could it happen that it isn't?

Reply:
It's always the other person https://github.com/decentraland/matrix-client/blob/0d3614621c8020201da2e75b46b2daf9aff18ebe/src/MessagingClient.ts#L65